src/frontend/get-jira-configuration.ts

```diff
@@ -56,14 +56,14 @@ export const getFailedConnections = (installations, subscriptions) => {
 		.map((failedConnection) => ({
 			...failedConnection,
 			...subscriptions.find(
-				(sub) =>
-					failedConnection.id === sub.gitHubInstallationId
+				(sub) => failedConnection.id === sub.dataValues.gitHubInstallationId
 			),
 		}))
 		.map((sub) => {
-			const repos = sub.reposSyncState || {};
+			const repos = sub?.dataValues?.repoSyncState?.repos || {};
```

Comment:
don't use `dataValues` unless there's a reason for it.

Reply:
Not using dataValues is what caused this to break the last time I addressed PR comments.